packages/frontend/tailwind.config.ts

```diff
@@ -14,6 +14,7 @@ const config = {
     },
     extend: {
       screens: {
+        sm: '640px',
         ml: '820px',
```

Comment:
Modify the default screen sizes based on tailwind's recommendation
https://tailwindcss.com/docs/responsive-design